src/jdk.jdwp.agent/share/native/libjdwp/invoker.h

```diff
@@ -35,7 +35,7 @@
 typedef struct InvokeRequest {
     jboolean pending;      /* Is an invoke requested? */
     jboolean started;      /* Is an invoke happening? */
-    jboolean available;    /* Is the thread in an invokable state? */
+    jboolean available;    /* Is the thread in an invocable state? */
```

Comment:
invocable could perhaps stay as invokable ?
Elsewhere we have a filename com/sun/tools/jdi/InvokableTypeImpl.java which we clearly don't want to change,  and I see Internet dictionary evidence of invokable being acceptable.

Reply:
Sure, I just thought there was enough evidence that invokable is not definitely wrong, even if invocable is more correct and popular, so it's not obvious it should be changed.  Don't lose sleep over it. 8-)

More importantly, on the tests -- I see the changes in exception messages searched for the wrong text in the test directories, and didn't find any matches that looked like checks.

Reply:
Invocable, Invokable and Invokeable are all used in practice. We have a mix of invocable and invokable throughout our codebase, with more of the former than the latter - and in prose "invocable" is probably best.